client/apps/edit/components/content/sections/hero/index.jsx

```diff
@@ -1,53 +1,75 @@
+import Backbone from 'backbone'
 import PropTypes from 'prop-types'
 import React, { Component } from 'react'
 import SectionContainer from '../../section_container/index.coffee'
 import SectionTool from '../../section_tool/index.jsx'
 
-export default class SectionHero extends Component {
+export class SectionHero extends Component {
+  static propTypes = {
+    article: PropTypes.object.isRequired,
+    channel: PropTypes.object,
+    onChange: PropTypes.func.isRequired
+  }
+
   constructor (props) {
     super(props)
-    this.state = { editing: false }
+
+    const hero = new Backbone.Model(props.article.get('hero_section'))
```

Comment:
Same here, these are technically "sections" so we should be able to use that model otherwise we might want to create a separate one?